app/src/components/market/common/history_chart/index.tsx

```diff
@@ -29,7 +28,7 @@ const ChartTooltip = styled.div`
 `
 
 const TooltipTitle = styled.h4`
-  color: ${props => props.theme.colors.textColorDark};
+  color: ${props => props.theme.text1};
```

Comment:
Man we went through this on call this morning and I left feedback on last pr. Not sure why is this still there?

Reply:
if you were less aggressive in criticism it might make for a more positive work environment, which may lead to less stressed workers and less mistakes. this is not how you treat someone who is new to a code base.

Reply:
@gem1no I try to be patient and polite but Im not perfect I have certain thresholds. You may be right Im a little rough around the edges sometimes. But at least Im honest and you can always know where you stand with me and I speak from my heart I don't lie or try to deceive. Looking the way you left and how you are talking shit now that you are not here tells me a lot about you and that it maybe was a good decision we made since you clearly weren't honest with all of us and that is coming out now. In my life I have grown the most with people who were honest with me then with people who were superficially polite, not everyone appreciates that. https://media.giphy.com/media/qEfGLWRq9LokP6cBeX/giphy.gif

Reply:
@gem1no no. @Mi-Lan has been very supportive, helpful, and honest. We have had quite some misunderstanding on this issue and that's 95% my fault. I am starting here and I have not taken the right attitude towards timing, quality, and research, I am fully aware and changing this. The remote and independent contribution working environment is very much like a double hedge sword and requires adaptation but mostly learning and honesty.

I have done all my commits in the way that seemed most coherent and in good faith. Though I went too fast and agitated. That will change.